(on|off)boarding: add wiki entries (#27)
NixOS/moderation: boarding.md

  - [ ] Invite into the Moderation Team organization (via Infrastructure team)
- Email
  - [ ] Receive email for the moderation@ alias (via Infrastructure team)
- Wiki

>   - [ ] [Request the `moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator), if desired

Reviewer: ```suggestion
  - [ ] Request acces to the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator), if desired
```

Let's only link that part.

Author: Definitely. Fixed in latest push.

>   - [ ] Request the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator), if desired

Reviewer: ```suggestion
  - [ ] Request access to the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator)
```

  - [ ] Rotate secrets
- Email
  - [ ] Remove from moderation@ alias (via Infrastructure team)
- Wiki

>   - [ ] Remove the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator)

Reviewer: ```suggestion
  - [ ] Remove from the [`moderator` user group](https://wiki.nixos.org/wiki/NixOS_Wiki:Moderator)
```
